csv-res-filter/pom.xml

```diff
@@ -2,14 +2,14 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.ibm.g11n.pipeline</groupId>
-	<artifactId>csv-filter</artifactId>
+	<artifactId>csv-res-filter</artifactId>
```

Comment:
Please look at other modules in this repository.
Because we want to include this one as a part of gp-java-tools release, we want to share the same parent pom - 

```
	<parent>
		<groupId>com.ibm.g11n.pipeline</groupId>
		<artifactId>gp-java-tools</artifactId>
		<version>1.2.7-SNAPSHOT</version>
	</parent>
```

Reply:
added